pagopa/io-app
# feat(Firma con IO): [SFEQS-1661] Edit signature fields order

## Short description
This PR changes the signature fields order during a signing flow.
The new order is: 
1. UNFAIR;
2. REQUIRED;
3. OPTIONAL.

## List of changes proposed in this pull request
- Adds a `orderSignatureFields` function which returns a new ordered readonly array;
- Calls the aformentioned function when rendering the signature fields in `FciSignatureFieldsScreen.tsx`;
- Updates the signature fields rendering test accordingly.

## How to test
- Start the dev server with the following configuration:
```
fci: {
      waitForSignatureCount: 1,
      rejectedCount: 0,
      expiredCount: 0,
      expired90Count: 0,
      waitForQtspCount: 0,
      signedCount: 0,
      noSignatureFieldsCount: 0
    },
```
- Start the app by pointing to the dev server and start a signature flow from the message;
- Press `Go to the documents` and then press `Continue` and `Go to signature`;
- Signature fields should follow the aformentioned order.
